modules/virt-node-placement-rule-examples.adoc

```diff
@@ -152,16 +152,16 @@ spec:
 [id="hostpathprovisioner-object-node-placement-rules_{context}"]
 == HostPathProvisioner object node placement rule example
 
-You can edit the `HostPathProvisioner` object directly or by using the web console. 
+You can edit the `HostPathProvisioner` object directly or by using the web console.
 
 [WARNING]
 ====
 You must schedule the hostpath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner cannot run. You cannot run virtual machines.
 ====
 
-If after deploying a virtual machine (VM) with the hostpath provisioner (HPP) storage class you remove the hostpath provisioner pod from the same node by using the node selector, you must first revert the change, at least for that specific node, and wait for the pod to run before trying to delete the VM.
+After you deploy a virtual machine (VM) with the hostpath provisioner (HPP) storage class, you can remove the hostpath provisioner pod from the same node by using the node selector. However, you must first revert that change, at least for that specific node, and wait for the pod to run before trying to delete the VM.
```

Comment:
The shorter sentences are better but they are in the reverse order. If the user must first revert the change (explain in more detail perhaps) and THEN remove the HPP (use the abbreviation, since you used it earlier) pod from the node. 

Example: To remove a VM with HPP storage class:

1. Update the node by reverting the change (whatever that means -- something to do with the node selector)
2. Wait for the HPP pod to start up again.
3. Delete the VM.

For usability reasons, we try to keep actions that must be performed in a certain order in the order that the user must follow.

Reply:
@dalia-frank, is the order that Avital suggests correct? I thought that the user would do what is said in sentence 1. Then if the user wants to later delete the VM, the user has to revert the change for the specific node and wait for the pod to run.